frontend/js/src/playlists/CreateOrEditPlaylistModal.tsx

```diff
@@ -200,66 +188,29 @@ export default class CreateOrEditPlaylistModal extends React.Component<
               </div>
 
               <div className="form-group">
-                <label htmlFor="playlistcollaborators">Collaborators</label>
-                <table
-                  id="playlistcollaborators"
-                  className="table table-condensed table-striped listens-table"
-                >
-                  <tbody>
-                    {collaborators.map((user, index) => {
-                      return (
-                        <tr key={user}>
-                          <td>{user}</td>
-
-                          <td
-                            style={{
-                              paddingTop: 0,
-                              paddingBottom: 0,
-                              width: "30px",
-                            }}
-                          >
-                            <button
-                              className="btn btn-link"
-                              type="button"
-                              aria-label="Remove"
-                              onClick={this.removeCollaborator.bind(this, user)}
-                            >
-                              <FontAwesomeIcon icon={faTimes as IconProp} />
-                            </button>
-                          </td>
-                        </tr>
-                      );
-                    })}
-                  </tbody>
-                </table>
-                <div className="input-group input-group-flex">
-                  <span className="input-group-addon">Add collaborator</span>
-                  <input
-                    type="text"
-                    className="form-control"
-                    placeholder="Username…"
-                    name="newCollaborator"
-                    onChange={this.handleInputChange}
-                    value={newCollaborator}
-                  />
-                  <span className="input-group-btn">
-                    <button
-                      className="btn btn-default"
-                      type="submit"
-                      onClick={this.addCollaborator}
-                      disabled={
-                        !newCollaborator ||
-                        (isEdit
-                          ? playlist?.creator.toLowerCase() ===
-                            newCollaborator.toLowerCase()
-                          : currentUser.name.toLowerCase() ===
-                            newCollaborator.toLowerCase())
```

Comment:
This bit of logic would now be removed, and needs to be implemented in `addCollaborator`.
It prevents adding to the collaborators the creator of the playlist (for an existing playlist, see how `isEdit` is defined above), *or* the currently logged-in user (for a new playlist being created).

The reason for that is that a user should not be able to be in the collaborators if they are the owner of the playlist.